File: WorkRelated/IeWebDriver.java
//fixed sample code from:
//https://docs.microsoft.com/en-us/microsoft-edge/webdriver-chromium/ie-mode?tabs=java#known-limitations

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.ie.InternetExplorerDriver;
import org.openqa.selenium.ie.InternetExplorerOptions;

public class IEDriverSample {
    public static void main(String[] args) {
    
        InternetExplorerOptions ieOptions = new InternetExplorerOptions();
        ieOptions.attachToEdgeChrome();
        ieOptions.withEdgeExecutablePath("C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe");
        System.setProperty("webdriver.ie.driver", "C:\\IEDriverServer\\IEDriverServer.exe");
        
        WebDriver driver = new InternetExplorerDriver(ieOptions);
        driver.get("http://www.bing.com");
    }
}

File: WorkRelated/IeWebDriver.cs
//fixed sample code from:
//https://docs.microsoft.com/en-us/microsoft-edge/webdriver-chromium/ie-mode?tabs=c-sharp#known-limitations

using System;
using OpenQA.Selenium;
using OpenQA.Selenium.IE;

namespace IEDriverSample
{
    class Program
    {
        static void Main(string[] args)
        {
            var ieOptions = new InternetExplorerOptions();
            ieOptions.AttachToEdgeChrome = true;
            //change the path accordingly
            ieOptions.EdgeExecutablePath = "C:/Program Files (x86)/Microsoft/Edge/Application/msedge.exe";

            var driver = new InternetExplorerDriver("C:\\IEDriverServer\\", ieOptions);
            driver.Url = "https://bing.com";

        }
    }
}
